test/bigtest/tests/find-package-test.js

```diff
@@ -231,4 +235,199 @@ describe('ui-plugin-find-package-title', function () {
       });
     });
   });
+
+  describe('when the plugin is rendered with a multiselect prop', () => {
+    beforeEach(async function () {
+      onRecordChosenHandler.resetHistory();
+
+      await renderComponent({
+        isMultiSelect: true,
+      });
+    });
+
+    describe('and a trigger button is clicked', () => {
+      beforeEach(async () => {
+        await plugin.triggerButton.click();
+      });
+
+      it('should open the plugin modal', () => {
+        expect(plugin.modalIsDisplayed).to.be.true;
+      });
+
+      describe('when searching for packages', () => {
+        beforeEach(async () => {
+          await plugin.modal.searchField.fill('Selected');
+          await plugin.modal.clickSelectedFilter();
+        });
+
+        describe('and submit button was clicked', () => {
+          beforeEach(async () => {
+            await plugin.modal.searchButton.click();
+          });
+
+          it('should display a list of loaded packages', () => {
+            expect(plugin.modal.resultsList.rows().length).to.equal(2);
+          });
+
+          describe('and a package was selected', () => {
+            beforeEach(async () => {
+              await plugin.modal.resultsList.rows(0).click();
+            });
+
+            it('should not close the modal', () => {
+              expect(plugin.modalIsDisplayed).to.be.true;
+            });
+
+            it('should update titles selected count in the footer', () => {
+              expect(plugin.modal.titlesSelectedText).to.equal('Titles selected: 1');
+            });
+
+            describe('and clicking Cancel', () => {
+              beforeEach(async () => {
+                await plugin.modal.cancelSelection();
+              });
+
+              it('should close the modal', () => {
+                expect(plugin.modalIsDisplayed).to.be.false;
+              });
+
+              it('should not call the provided callback', () => {
+                expect(onRecordChosenHandler.callCount).to.equal(0);
+              });
+            });
+
+            describe('and selecting a second package', () => {
+              beforeEach(async () => {
+                await plugin.modal.resultsList.rows(1).click();
+              });
+
+              describe('and clicking Save', () => {
+                beforeEach(async () => {
+                  await plugin.modal.saveSelection();
+                });
+
+                it('should close the modal', () => {
+                  expect(plugin.modalIsDisplayed).to.be.false;
+                });
+
+                it('should call the provided callback with correct data', () => {
+                  expect(onRecordChosenHandler.calledOnceWith([
```

Comment:
I see you use the same object in config file. Please import it here form config. It could help to omit duplicate in code